### DISCUSSION TITLE
📎 VSCode extension option to apply quick fixes on save

### DISCUSSION THREAD

**Discussion by jpike88** [rome/tools#4162] [Suggestions]
### Description

I'd prefer quick fixes to be applied each time I save, I enjoyed this function with eslint.

**Comment by ematipico** [comment]
I believe this has been answered already: https://github.com/rome/tools/discussions/3856#discussioncomment-4237754

**Reply by jpike88** [reply]
It doesn't provide a tracking issue, so I don't know if it's been implemented yet or not. I'm on the prerelease vscode extension, and it has `Connecting to "/var/folders/0v/w61lqyb97l37zrp46x8s24900000gn/T/rome-socket-11.0.0"` in the Output terminal.

Am I supposed to just wait or can I follow something?

**Reply by ematipico** [reply]
Rome v11 supports this, which is the latest version available on npm. You'd need to change your `settings.json` in VSCode and add `"editor.codeActionsOnSave": { "quickfix.rome": true }`.

If that doesn't work, I'd suggest youthe support channel on discord or open an issue with the info of the `rome rage` command and the daemon logs

**Reply by jpike88** [reply]
ah I see, the option just isn't visible via the vscode settings pane, I have to manually edit settings.json.

works well (aside from some rules quick fix being wonky, I've logged them)

**Reply by ianzone** [reply]
> Rome v11 supports this, which is the latest version available on npm. You'd need to change your `settings.json` in VSCode and add `"editor.codeActionsOnSave": { "quickfix.rome": true }`.
> 
> If that doesn't work, I'd suggest youthe support channel on discord or open an issue with the info of the `rome rage` command and the daemon logs

this need to be listed in the plugin descriptions, otherwise people wouldn't know.

**Reply by ematipico** [reply]
PRs are welcome!

**Reply by ianzone** [reply]
> "editor.codeActionsOnSave": { "quickfix.rome": true }

here you are https://github.com/rome/tools/pull/4339, but a better way of setting is still via UI.
